### DISCUSSION TITLE
Possible to add Cricket?

### DISCUSSION THREAD

**Discussion by solyxius** [vasqued2/ha-teamtracker#68] [Ideas]
Firslty, thanks for your awesome work @vasqued2 !!!

Any chance of tapping into the espncricinfo.com api's as mentioned here (https://github.com/outside-edge/python-espncricinfo)?

Seemingly we'd need to be able to specify an alternate URL ...

Cheers for your consideration

**Comment by vasqued2** [comment]
It currently supports the ESPN cricket APIs as of v0.6.1 with the caveats called out in the Wiki.

[https://github.com/vasqued2/ha-teamtracker/wiki/Setting-up-a-Sensor-for-Cricket](https://github.com/vasqued2/ha-teamtracker/wiki/Setting-up-a-Sensor-for-Cricket
)

**Reply by solyxius** [reply]
Doh, I was so excited when I found your repo, I forgot to do some reading!!!

Thanks David.
